backend/users/views.py

```diff
@@ -1,4 +1,29 @@
+from django.contrib.auth import login
+from django.contrib.auth.views import LoginView
 from django.shortcuts import render  # noqa
+from django.urls import reverse_lazy
+from django.views.generic.edit import FormView
 
+from .forms import LoginForm, SignUpForm
 
-# Create your views here.
+
+class Login(LoginView):
```

Comment:
To avoid losing the `LoginView` name here, you can import Django's LoginView like this: `from django.contrib.auth.views import LoginView as DjangoLoginView`

Then you'll be able to call your own view `LoginView`.